Partition (WBP)/Functions/V57033F.VT157033.fun.yml

```diff
@@ -183,555 +272,3640 @@
     Add_Entry To_List(#wLTestResultExactSigned_Decimals)
     
     * *** Generic Signed Results
-    #LF_Test_Result_L157033FI1 #LF_Test_Result_L157033FN := 320
-    #LF_Test_Result_L157033FI2 #LF_Test_Result_L157033FI4 #LF_Test_Result_L157033FI8 #LF_Test_Result_L157033FP := 0
-    #LF_Test_Result_L157033FPD #LF_Test_Result_L157033FND := 320
+    #LF_Test_Result_L157033FP := 0
+    #LF_Test_Result_L157033FI1 #LF_Test_Result_L157033FI2 #LF_Test_Result_L157033FI4 #LF_Test_Result_L157033FI8 #LF_Test_Result_L157033FN #LF_Test_Result_L157033FND #LF_Test_Result_L157033FPD := 320
     Add_Entry To_List(#wLTestResultGenericSigned)
     * BACKWARDS
-    #LF_Test_Result_L157033FI1 #LF_Test_Result_L157033FN := 336
+    #LF_Test_Result_L157033FI1 #LF_Test_Result_L157033FI2 #LF_Test_Result_L157033FI4 #LF_Test_Result_L157033FI8 #LF_Test_Result_L157033FN := 336
     #LF_Test_Result_L157033FPD #LF_Test_Result_L157033FND := 321
     Add_Entry To_List(#wLTestResultGenericSignedBackwards)
+    
     * FORWARDS DESCENDING
-    #LF_Test_Result_L157033FI1 #LF_Test_Result_L157033FN := 672
+    #LF_Test_Result_L157033FI1 #LF_Test_Result_L157033FI2 #LF_Test_Result_L157033FI4 #LF_Test_Result_L157033FI8 #LF_Test_Result_L157033FN := 672
     #LF_Test_Result_L157033FPD #LF_Test_Result_L157033FND := 642
     Add_Entry To_List(#wLTestResultGenericSignedForwardsDescending)
-    
+    * BACKWARDS DESCENDING
+    #LF_Test_Result_L157033FI1 #LF_Test_Result_L157033FI2 #LF_Test_Result_L157033FI4 #LF_Test_Result_L157033FI8 #LF_Test_Result_L157033FND #LF_Test_Result_L157033FPD #LF_Test_Result_L157033FND := 320
+    #LF_Test_Result_L157033FP := 336
+    Add_Entry To_List(#wLTestResultGenericSigned)
```

Comment:
You are using the same list as on line 277. Thats incorrect. This is still search <1>. Each list only used once per search. I suspect that the entire set of results for BACKWARDS DESCENDING is the same as FORWARDS and hence this whole set of initialisation should be deleted and '#LF_Test_Result_L157033FP := 336' used on line 275.

Reply:
Ok..I will look into the above feedback.

Reply:
@robe070 , yes. BACKWARDS DESCENDING is the same as FORWARDS in almost all cases . In this case, Can we have 3 lists For Ex:For generic signed results, lists will be:                                                                                                           1) FORWARDS [#wLTestResultGenericSigned]                     
2) BACKWARDS [#wLTestResultGenericSignedBackwards]                                                    
3) FORWARDS DESCENDING [#wLTestResultGenericSignedForwardsDescending]

Reply:
Its not a 'can' question. It depends on the results of a query. As I've posted "Do not duplicate an entire set of results in a new list"

Reply:
Search<1> results updated including all numeric. Kindly review .PFA 
[Verifier_Test_Report.txt] (https://github.com/lansa/db-regression/files/13784701/Verifier_Test_Report.txt)
fier report .txt

Reply:
Will complete the remaining Search<2> - Search<8>

Reply:
Search2 testresults- Working successfully on both IBMi and windows .PFA verifier report. Kindly review.
[Verifier_Test_Report.txt](https://github.com/lansa/db-regression/files/13790972/Verifier_Test_Report.txt)

Reply:
So far so good.